zeppelin-web/src/components/navbar/navbar.controller.js

```diff
@@ -90,22 +90,32 @@ function NavCtrl ($scope, $rootScope, $http, $routeParams, $location,
 
     // for firefox and safari
     logoutURL = logoutURL.replace('//', '//false:false@')
-    $http.post(logoutURL).error(function () {
-      // force authcBasic (if configured) to logout
-      $http.post(logoutURL).error(function () {
+
+    let config = (process.env.PROD) ? {headers: { 'X-Requested-With': 'XMLHttpRequest' }} : {}
+    $http.post(logoutURL, config).then(
+      function (response) {
+        alert('ok response status ', response.status)
```

Comment:
I'm not sure we should have a blocking dialog box here?

Reply:
I can remove it of course, this was simply the original behavior... just tell me what to do :-)

Reply:
argh sorry, haven't noticed

Reply:
removed, you were correct, thanks!